.gitignore

```diff
@@ -26,3 +27,6 @@ CSharp/avayacloud-client-common/avayacloud-client-common.xml
 CSharp/avayacloud-client-subscription/avayacloud-client-subscription.xml
 *.user
 CSharp/avayacloud-client-subscription/packages.config
+
```

Comment:
Please double check this file with the latest on master. I believe nothing is needed here.